.circleci/config.yml

```diff
@@ -2,51 +2,21 @@
 
 version: 2.1
 
-anchors:
-  install_yarn_deps: &install_yarn_deps
-    run:
-      name: Install Dependencies
-      command: yarn install --immutable
-
-  restore_yarn_cache: &restore_yarn_cache
-    restore_cache:
-      name: Restore Yarn Package Cache
-      keys:
-        - yarn-packages-v2-{{ arch }}-{{ checksum "yarn.lock" }}
-        - yarn-packages-v2-{{ arch }}
-
-  save_yarn_cache: &save_yarn_cache
-    save_cache:
-      name: Save Yarn Package Cache
-      key: yarn-packages-v2-{{ arch }}-{{ checksum "yarn.lock" }}
-      paths:
-        - ./node_modules
-
-  build: &build
-    run:
-      name: Build Add-On
-      command: yarn build
-
-  lint: &lint
-    run:
-      name: ESLint
-      command: |
-        yarn lint
-
-  test: &test
-    run:
-      name: Integration/Unit Tests
-      command: |
-        yarn test
+orbs:
+  node: circleci/node@5.1
 
 jobs:
   build:
-    docker:
-      - image: cimg/node:12.14
+    executor: node/default
     steps:
       - checkout
-      - *restore_yarn_cache
-      - *install_yarn_deps
-      - *save_yarn_cache
-      - *build
-      - *test
+      - node/install:
+          install-yarn: true
```

Comment:
This config works the same as before.

Except it'll get the Node version from `.nvmrc`, and we won't have to change it if we bump the Node version.